src/impact-dashboard/ChartArea.tsx

```diff
@@ -25,22 +36,74 @@ const LegendContainer = styled.div`
   flex: 0 0 auto;
 `;
 
-const ChartArea: React.FC = () => {
+function combinePopulations(data: CurveData, columnIndex: number) {
```

Comment:
This point might be moot if you follow @macfarlandian's advice to move the calculation logic back to where it was previously. But if it does get moved out for some reason, we should try to not have two identical definitions of this function in two separate places.

Reply:
this function is not needed here, just forgot to delete it, but I think it's better for the parent component to hold the data, makes it a lot easier to share data between different children, instead of sibling communication, I put this data initially on it's own context. but as per my discussion with Ian, it may seem like not needed at the moment but it will make things a lot easier going forward.